src/components/BrandedFooter.tsx

```diff
@@ -10,6 +10,7 @@ import ExternalLink from './ExternalLink'
 const UniswapA = styled(ExternalLink)`
   color: ${({ theme }) => theme.secondary};
   cursor: pointer;
+  margin-top: 12px;
```

Comment:
The anchor itself should not have a margin-top. It should be a part of the containing element (eg as a gap), either of the Row of the BrandedFooter, or of the Footer's containing element.